src/widgets/AutocompleteSelector.js

```diff
@@ -93,12 +93,11 @@ export class AutocompleteSelector extends React.PureComponent {
       <View style={localStyles.container}>
         <SearchBar
           autoCapitalize="none"
-          autoCorrect={false}
+          autoCorrect={true}
```

Comment:
Why has this been changed? Because we handle a lot of different countries, correcting spelling can be a little bit of a nightmare.. which countries spelling to use etc.. I think it might be best to keep this as `false`?

Reply:
OK, cool. It's logical to set it as "true" isn't it? but anyway I have implemented it in the branch.

Reply:
no - as explained auto correction with fields that are not standard words is not good